operations/patch_util.go

```diff
@@ -454,8 +454,16 @@ func (p *patchParams) loadTasks(conf *ClientSettings) error {
 				return errors.Wrap(err, "setting default tasks")
 			}
 		}
+		if len(defaultTasks) > 0 {
+			grip.Info("Using default tasks set in local config")
```

Comment:
nit: can be combined into `grip.InfoWhen(len(defaultTasks) > 0, "Using default tasks set in local config")`.

Reply:
ty